tests/Data/ProgrammesDb/EntityRepository/ServiceRepository/FindByIdsWithNetworkServicesListTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+declare(strict_types=1);
+
+namespace Tests\BBC\ProgrammesPagesService\Data\ProgrammesDb\EntityRepository\ServiceRepository;
+
+use BBC\ProgrammesPagesService\Data\ProgrammesDb\EntityRepository\ServiceRepository;
+use BBC\ProgrammesPagesService\Domain\ValueObject\Sid;
+use Tests\BBC\ProgrammesPagesService\AbstractDatabaseTest;
+
+/**
+ * @covers BBC\ProgrammesPagesService\Data\ProgrammesDb\EntityRepository\ServiceRepository::<public>
+ */
+class FindByIdsWithNetworkServicesListTest extends AbstractDatabaseTest
+{
+    /** @var  ServiceRepository */
```

Comment:
TOO MANY SPACES!!!!!

Reply:
jesus, i'm sorry, i'm sorry!

Reply:
Don't be sorry to Jesus. Be sorry to me.